flixel/input/mouse/FlxMouseEventManager.hx

```diff
@@ -38,6 +32,17 @@ import flixel.util.FlxDestroyUtil;
  * function onMouseOver(object:FlxObject) {}
  * function onMouseOut(object:FlxObject) {}
  * ```
+ * 
```

Comment:
Doc below this comment probably should be moved to `FlxMouseEvent`, but let it be here for now.

Reply:
Why not do it now?